solr/core/src/java/org/apache/solr/metrics/otel/instruments/AttributedInstrumentFactory.java

```diff
@@ -36,16 +34,19 @@
  * Factory for creating metrics instruments that can write to either single or dual registries (core
  * and node).
  */
+// TODO consider making this a base abstraction with a simple impl and another "Dual" one.
 public class AttributedInstrumentFactory {
 
+  // These attributes are on a core but don't want to aggregate them.
   private static final Set<AttributeKey<?>> FILTER_ATTRS_SET =
-      Set.of(COLLECTION_ATTR, CORE_ATTR, SHARD_ATTR, REPLICA_TYPE_ATTR, INTERNAL_ATTR);
```

Comment:
There were too many here IMO; we just want to exclude the ones identifying in nature.  Now we can differentiate aggregate between internal vs not -- very useful.